index.html

```diff
@@ -654,5 +654,368 @@ <h2>Exposing change of MediaStreamTrack configuration</h2>
+      <h3>{{VideoFrameMetadata}}</h3>
+      <pre class="idl">
+partial dictionary VideoFrameMetadata {
+  sequence&lt;Segment&gt; segment;
+};</pre>
+      <section class="notoc">
+        <h4>Members</h4>
+        <dl class="dictionary-members" data-link-for="VideoFrameMetadata" data-dfn-for="VideoFrameMetadata">
+          <dt><dfn><code>segment</code></dfn> of type <code>sequence&lt;{{Segment}}&gt;</code></dt>
+          <dd>
+            <p>The set of known geometrical segments in the video frame.</p>
+          </dd>
+        </dl>
+      </section>
+    </section>
+    <section>
+      <h3>{{Segment}}</h3>
+      <pre class="idl">
+dictionary Segment {
+  required SegmentType type;
+  required long        id;
+  required long        partOf;
+  required float       probability;
+  Point2D              centerPoint;
+  DOMRectReadOnly      boundingBox;
+};</pre>
+<pre class="idl">
+enum SegmentType {
+  "human-face",
+  "left-eye",
+  "right-eye",
+  "eye",
+  "mouth",
+};</pre>
+      <section class="notoc">
+        <h4>Dictionary {{Segment}} Members</h4>
+        <dl class="dictionary-members" data-dfn-for="Segment" data-link-for="HumanFace">
+          <dt><dfn><code>type</code></dfn> of type <code>{{SegmentType}}</code></dt>
+          <dd>
+            <p>The type of segment which the segment refers to.</p>
+            <p>It must be one of the following values:</p>
+            <dl data-dfn-for="SegmentType" data-link-for="SegmentType">
+              <dt><dfn><code>human-face</code></dfn></dt>
+              <dd>
+                <p>The segment describes a human face.</p>
+              </dd>
+              <dt><dfn><code>left-eye</code></dfn></dt>
+              <dd>
+                <p>The segment describes <i>oculus sinister</i>.</p>
+              </dd>
+              <dt><dfn><code>right-eye</code></dfn></dt>
+              <dd>
+                <p>The segment describes <i>oculus dexter</i>.</p>
+              </dd>
+              <dt><dfn><code>eye</code></dfn></dt>
+              <dd>
+                <p>The segment describes an eye, either left or right.</p>
+              </dd>
+              <dt><dfn><code>mouth</code></dfn></dt>
+              <dd>
+                <p>The segment describes a mouth.</p>
+              </dd>
+            </dl>
+          </dd>
+          <dt><dfn><code>id</code></dfn> of type {{long}}</dt>
+          <dd>
+            <p>An identifier of the segment, unique within a sequence. If the same segment can be tracked over multiple
+            frames originating from the same {{MediaStreamTrack}} source, the user agent SHOULD use the same {{Segment/id}}
+            for the segment in all frames. {{Segment/id}} is also used in conjunction with the member {{Segment/partOf}}.</p>
+            <p>The user agent MUST NOT select the value to assign to {{Segment/id}} in such a way that the detected segments
+            could be correlated to match between different {{MediaStreamTrack}} sources.</p>
```

Comment:
What about MediaStreamTrack clones?

Reply:
My thinking was that any MediaStreamTrack clones have "frames originating from the same {{MediaStreamTrack}} source". Is that not true? I will clarify the PR and mention MediaStreamTrack clones explicitly.